runtime/oti/j9port_generated.h

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 1991, 2019 IBM Corp. and others
+ * Copyright (c) 1991, 2020 IBM Corp. and others
```

Comment:
Please undo this change - the file is otherwise unmodified.

Reply:
Fixed in [force-push](https://github.com/eclipse/openj9/compare/e200fcd0089727bb8c11ffa2d51bc48880865951..a15fd0a5f1583c638085002e198a4ef9732245cf).